wiki/fr/fr-Installation-for-Macintosh.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: wiki
+title: "Installation sous MacOS"
+lang: "fr"
+permalink: "/wiki/Installation-for-Macintosh"
+---
+
+# Installation sous MacOS
+
+Assurez-vous de lire le [guide de démarrage et installation](Getting-Started) avant de commencer.
+
+1. [Téléchargez Jamulus](https://sourceforge.net/projects/llcon/files/latest/download){: target="_blank" rel="noopener noreferrer" .button}
+1. **Décompressez le fichier .zip téléchargé**. Double-cliquez sur le fichier `.zip`, ce qui créera un nouveau dossier du même nom. Le dossier contient le fichier de licence et un fichier `.dmg` contenant l'application.
+1. **Double-cliquez sur le fichier `.dmg`** pour l'ouvrir. L'image s'ouvre et vous verrez deux applications (client et serveur Jamulus).
+1. **Glissez et déposez les deux icônes dans le dossier Applications** pour installer Jamulus.
+1. **Executez Jamulus**. Vous devriez maintenant pouvoir utiliser Jamulus comme n'importe quelle autre application. Si vous voyez un message qui vous dit que Jamulus ne peut pas être ouvert parce que le développeur ne peut pas être vérifié, lisez la suite et jetez un coup d'oeil à la [section suivante](#jamulus-ne-peut-pas-être-ouvert-car-le-développeur-ne-peut-pas-être-vérifié).
+
+_Vous pouvez supprimer le dossier dans le répertoire Downloads contenant le `.dmg` et éjecter le lecteur "Jamulus" de votre bureau. Ils ne sont plus nécessaires._
+
+Vous pouvez également [compiler vous même le logiciel](Compiling) si vous en avez envie.
+
+## "Jamulus" ne peut pas être ouvert car le développeur ne peut pas être vérifié
```

Comment:
"_lancé_" rather than "_ouvert_" maybe? or even better to keep consistency in the page: "_exécuté_"?

Reply:
In the MacOS French UI you must clic on « ouvrir » on the contextual menu. So it make sense to keep as it and say « ne peut pas être ouvert ».

Reply:
And it is exactly the message that the user see.
https://support.apple.com/fr-fr/guide/mac-help/mh40616/mac

Reply:
I think we could close this point and resolve the conversation? Everyone agree?

Reply:
If that is the case under MacOs. No worries then.